src/flex-grid/types.js

```diff
@@ -17,4 +17,9 @@ export type FlexGridPropsT = {
   flexGridRowGap?: ResponsiveT<ScaleT>,
 } & BlockPropsT;
 
-export type FlexGridItemPropsT = FlexGridPropsT;
+export type FlexGridItemPropsT = {
```

Comment:
one last thing before merging - can you also update the typescript definitions?

Reply:
Updated!